lib/pushmi_pullyu/cli.rb

```diff
@@ -104,6 +104,10 @@ def parse_options(argv)
         opts[:piddir] = piddir
       end
 
+      o.on('-W', '--workdir PATH', 'Path where downloads, etc. are done ') do |workdir|
+        opts[:workdir] = workdir
```

Comment:
Derp, right. Maybe we should rename that whole thwack of constants. DEFAULT_PIDDIR, DEFAULT_WORKDIR, etc? It's easy to come away with the impression that those are constants for the thing instead of constants for the defaults of those things.